main.css

```diff
@@ -57,7 +57,7 @@ ol li::before {
     height: 30px;
```

Comment:
Why is this file move out of `static` folder?

Reply:
Moved back in `html` folder with proper folder structure